src/main/java/org/craftercms/studio/impl/v1/service/security/SecurityServiceImpl.java

```diff
@@ -190,13 +195,13 @@ public Authentication getCurrentAuthentication() {
     @ValidateParams
     public Map<String,Object> getUserProfile(@ValidateStringParam(name = "user") String user) throws ServiceLayerException, UserNotFoundException {
         Map<String, Object> toRet = new HashMap<String, Object>();
-        User u = securityProvider.getUserByIdOrUsername(-1, user);
+        UserTO u = securityProvider.getUserByIdOrUsername(-1, user);
         if (u != null) {
             toRet.put(KEY_USERNAME, user);
             toRet.put(KEY_FIRSTNAME, u.getFirstName());
             toRet.put(KEY_LASTNAME, u.getLastName());
             toRet.put(KEY_EMAIL, u.getEmail());
-            toRet.put(KEY_EXTERNALLY_MANAGED, u.isExternallyManaged());
+            toRet.put(KEY_EXTERNALLY_MANAGED, u.getExternallyManaged() > 0 ? true : false );
             String authenticationType = studioConfiguration.getProperty(SECURITY_TYPE);
```

Comment:
Auth type is the type of the provider that authenticated the user. It shouldn't be the current security type in config, since even if it was something like `ldap`, a user can be authenticated by `db` in the end. Maybe we don't need to return this property here since it's only used by the new APIs

Reply:
This is V1 service. Property is there as backward compatibility. Once we move all services to API 2 this will go away.